requirements.txt

```diff
@@ -30,3 +30,4 @@ six==1.15.0
 SQLAlchemy==1.4.1
 sqlparse==0.4.1
 Werkzeug==1.0.1
+drf-yasg
```

Comment:
Set a particular frozen version of `drf-yasg` that you guarantee that it works. Leaving it hanging like this means yasg can update and break something. Checking out the branch and installing it gave me `drf-yasg==1.20.0` You can check what you have by doing `pip freeze`